packages/next/README.md

```diff
@@ -1667,9 +1668,13 @@ These presets / plugins **should not** be added to your custom `.babelrc`. Inste
       "next/babel",
       {
         "preset-env": {},
+        "preset-react": {},
+        "preset-typescript": {},
+        "class-properties": {},
+        "object-rest-spread": {},
         "transform-runtime": {},
-        "styled-jsx": {},
-        "class-properties": {}
+        "transform-react-remove-prop-types": {},
```

Comment:
I don't think this should be configurable. Same for the rest of the newly added config values. They should have good defaults and not require any changes.

Reply:
agreed